src/modules/billing/mappers/transaction.js

```diff
@@ -96,7 +97,8 @@ const dbToModelMapper = createMapper()
   .map('volume').to('volume', volume => volume ? parseFloat(volume) : null)
   .map().to('agreements', mapDBToAgreements)
   .map(['billingVolume', 'endDate', 'season']).to('billingVolume',
-    (billingVolume, endDate, season) => billingVolume ? getBillingVolumeForTransaction({ billingVolume, endDate, season }) : null);
+    (billingVolume, endDate, season) => billingVolume ? getBillingVolumeForTransaction({ billingVolume, endDate, season }) : null)
```

Comment:
Same here with regards to defaulting to null vs defaulting to zero. I'm not sure which is "more correct".

Reply:
I think it's OK here because billingVolume is a child object which may or may not be present